### ISSUE TITLE
Predicts vector with (integer) labels.

### ISSUE THREAD

**Issue by kakatua26** [steven2358/sklearn-matlab#2]
Add Predicts vector with (integer) labels in regression

**Comment by steven2358** [comment]
Can you elaborate? E.g. which regressor class shoudl this be added to? And, can you point to a class in the original scikit-learn that implements this method?

**Comment by kakatua26** [comment]
predict_proba will return proba's value, how do i know which class that related to this proba value? there's predict that returns label or class (in number) in the scikit-learn lib.

**Comment by steven2358** [comment]
The current implementation of predict_proba will return the positive class probabilities.

You're right in that predict_proba does not provide the same functionality as in scikit-learn yet. It should be updated and extended. Feel free to contribute with code.
